File: 1_Periodo/Lista 13/C/L13_Q04.c
#include <stdio.h>
#include <stdlib.h>

int main(){
    int matriz[10][3];
    int i, j, cont1, cont2, vetor[3] = {0};

    for(i = 0; i< 10; i ++){
        for(j = 0; j < 3; j ++){
            printf("Aluno %d, prova %d: ", i + 1, j + 1);
            scanf(" %d", &matriz[i][j]);
        }
        system("cls");
    }

    for(i = 0; i< 10; i ++){
        cont1 = 0;
        cont2 = 0;
        for(j = 0; j < 3; j ++){
            if(cont1 < matriz[i][j]){
                cont1 = matriz[i][j];
                cont2 = j;
            }
        }
        switch (cont2)
        {
        case 0:
            vetor[0]++;
            break;
        case 1:
            vetor[1]++;
            break;
        case 2:
            vetor[2]++;
            break;
        default:
            break;
        } 
    }

    printf("%d alunos cuja a melhor nota foi a prova 1\n", vetor[0]);
    printf("%d alunos cuja a melhor nota foi a prova 2\n", vetor[1]);
    printf("%d alunos cuja a melhor nota foi a prova 3\n", vetor[2]);
}

File: 1_Periodo/Lista 13/Python/L13_Q04.py
matriz = []

for i in range(10):
    print(f"Informe as notas do {i+1}º aluno: ")
    linha = [float(input()) for i in range(3)]

melhorProva1, melhorProva2, melhorProva3 = 0, 0, 0

for linha in matriz:
    prova = linha.index(max(linha))
    if prova == 0:
        melhorProva1+=1
    elif prova == 1:
        melhorProva2+=1
    else:
        melhorProva3+=1

print(f"{melhorProva1} alunos foi melhor na prova 1")
print(f"{melhorProva2} alunos foi melhor na prova 2")
print(f"{melhorProva3} alunos foi melhor na prova 3")